docs/docs/howto/coordinator-ha.md

```diff
@@ -0,0 +1,70 @@
+# Coordinator High-Availability
+
+:::info
+Coordinator HA is only available in the enterprise edition of Contrast.
+<!-- TODO(burgerdev): link to documentation of enterprise edition -->
+:::
+
+This guide shows how to scale the Contrast Coordinator to more than one instance and make it highly available.
+
+## Workflow
+
+<!-- TODO(burgerdev): this how-to anticipates planned changes to readiness which are not yet committed. -->
```

Comment:
Implemented in https://github.com/edgelesssys/contrast/pull/1467

Reply:
Nice, as that is already approved we can delete the comment here.